jsk_fetch_robot/jsk_fetch_diagnosis/scripts/check_driver_boards.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python
```

Comment:
`scripts/check_driver_boards.py` -> `node_scripts/check_driver_boards_node.py`

Reply:
changed the directory name from scripts to node_scripts